odsclient/core.py

```diff
@@ -377,6 +409,17 @@ def get_download_url(self,
         """
         return "%s/explore/dataset/%s/download/" % (self.base_url, dataset_id)
 
+    def get_realtime_push_url(self,
+                              dataset_id,  # type: str
+                              ):
+        # type: (...) -> str
+        """
+
+        :param dataset_id:
+        :return:
+        """
+        return "%s/api/push/1.0/%s/realtime/push/" % (self.base_url, dataset_id)
```

Comment:
The hardcoded version number here makes me think that we'll have to manage this a little better in the future. But for now we can live with this.

Reply:
Yes, we could definitely have more distinctive logic. I think this is still better as omitting the version number like in `get_download_url` so the code won't break if a new API version is rolled out.

Reply:
agreed